google_drive/google_docs.py

```diff
@@ -40,11 +35,15 @@ class GoogleDocumentBlock(XBlock):
 
     embed_code = String(
         display_name="Embed Code",
-        help="Google provides an embed code for Drive documents with a variety of settings. From inside a Google Drive document, select Publish to the Web from within the File menuto get to your Embed Code with your desired settings.",
+        help=(
+            "Google provides an embed code for Drive documents. In the Google Drive document, "
+            "from the File menu, select Publish to the Web. Modify settings as needed, click "
+            "Publish, and copy the embed code into this field."
+        ),
         scope=Scope.settings,
         default=textwrap.dedent("""
             <iframe
-                src="https://docs.google.com/presentation/d/1B22AsMwG7jgE39vm1BuiJClir_JUz1q077cPEKJR2mI/embed?start=false&loop=false&delayms=3000"
+                src="https://docs.google.com/presentation/d/1x2ZuzqHsMoh1epK8VsGAlanSo7r9z55ualwQlj-ofBQ/embed?start=true&loop=true&delayms=10000"
```

Comment:
I assume this hash/id is a default document that we provide -- should we move this into a configuration setting so we don't have to update the code when it changes?